assignments/assignments_test.go

```diff
@@ -10,44 +10,34 @@ import (
 	"go.uber.org/zap"
 )
 
-const (
-	// gitHubTestOrg   = "autograder-test"
-	gitHubTestOrgID = 30462712
-)
-
-// To enable this test, please see instructions in the developer guide (dev.md).
-// You will also need access to the autograder-test organization; you may request
-// access by sending your GitHub username to hein.meling at uis.no.
+// To run this test, please see instructions in the developer guide (dev.md).
 
 func TestFetchAssignments(t *testing.T) {
+	qfTestOrg := os.Getenv("QF_TEST_ORG")
```

Comment:
Note to self. Make test helpers that are accessible from all packages to ensure consistent use of these variables and appropriate message logging.